src/panels/config/zwave/zwave-network.ts

```diff
@@ -42,157 +57,197 @@ export class ZwaveNetwork extends LitElement {
-              .hass=${this.hass}
-              domain="zwave"
-              service="heal_network"
-            >
-              Heal Network
-            </ha-call-service-button>
-            <ha-service-description
-              .hass=${this.hass}
-              domain="zwave"
-              service="heal_network"
-              ?hidden=${!this._showHelp}
-            ></ha-service-description>
-
-            <ha-call-service-button
-              .hass=${this.hass}
-              domain="zwave"
-              service="start_network"
-            >
-              Start Network
-            </ha-call-service-button>
-            <ha-service-description
-              .hass=${this.hass}
-              domain="zwave"
-              service="start_network"
-              ?hidden=${!this._showHelp}
-            >
-            </ha-service-description>
-
-            <ha-call-service-button
-              .hass=${this.hass}
-              domain="zwave"
-              service="stop_network"
-            >
-              Stop Network
-            </ha-call-service-button>
-            <ha-service-description
-              .hass=${this.hass}
-              domain="zwave"
-              service="stop_network"
-              ?hidden=${!this._showHelp}
-            >
-            </ha-service-description>
-
-            <ha-call-service-button
-              .hass=${this.hass}
-              domain="zwave"
-              service="soft_reset"
-            >
-              Soft Reset
-            </ha-call-service-button>
-            <ha-service-description
-              .hass=${this.hass}
-              domain="zwave"
-              service="soft_reset"
-              ?hidden=${!this._showHelp}
-            >
-            </ha-service-description>
-
-            <ha-call-service-button
-              .hass=${this.hass}
-              domain="zwave"
-              service="test_network"
-            >
-              Test Network
-            </ha-call-service-button>
-            <ha-service-description
-              .hass=${this.hass}
-              domain="zwave"
-              service="test_network"
-              ?hidden=${!this._showHelp}
-            >
-            </ha-service-description>
-
-            <ha-call-api-button .hass=${this.hass} path="zwave/saveconfig">
-              Save Config
-            </ha-call-api-button>
-          </div>
-        </ha-card>
+        ${this._networkStatus
+          ? html`
+              <ha-card class="content network-status">
+                <div class="details">
+                  ${this._networkStarting
+                    ? html`
+                        <paper-spinner
+                          active
+                          alt="Starting Network..."
+                        ></paper-spinner>
+                        Starting Z-Wave Network...<br />
+                        <small
+                          >This may take a while depending on the size of your
```

Comment:
This is kinda ugly, can you please add a new line after `>` and before the `<` at the end of this text

Reply:
oh yeah that's bad. sometimes Prettier makes things less pretty :)